### ISSUE TITLE
problem with permissions

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#412]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/411

**Originally posted by Skeletino (May 15, 2016):** 

After update to 1.73 version with all needed permissions present in pex file regular users can't create new shops, they only can use shops already created before.
Permissions that I added to permissions.yml are:
Â  Â  - shopkeeper.hire
Â  Â  - shopkeeper.player.trade
Â  Â  - shopkeeper.player.buy
Â  Â  - shopkeeper.player.normal
Â  Â  - shopkeeper.trade
Â  Â  - shopkeeper.entity.villager
Â  Â  - shopkeeper.player
Â  Â  - shopkeeper.list.own
and yet when player type "/shopkeeper trade" or "/shopkeeper" message from pex pops up. And on 1.72 version everything works fine.

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (May 15, 2016):** 

v1.73 shouldn't have changed anything permissions related.

I will try this later with PEX on my test server.
Quote:message from pex pops up

What message do you mean? PEX shouldn't print any message for shopkeepers commands. If you mean the 'no permission' message, that should be the one from shopkeepers itself.

            Edited May 16, 2016

**Comment by blablubbabcDEV** [comment]
**Originally commented by Skeletino (May 15, 2016):** 

[at]blablubbabc:
thanks for quick answer. Pex shows usual mesage in chat when player don't have permission "You don't have permission to do that".

**Comment by blablubbabcDEV** [comment]
**Originally commented by blablubbabc (May 15, 2016):** 

[at]Skeletino:
Okay, I tried it on my test server with your exact same permissions, and I was able to create player shops just fine.
As you mentioned commands: Creating player shops via command is disabled by default. Maybe you have regenerated the config. So make sure that you have set 'create-player-shop-with-command' to true in the config. Otherwise it might think that the player tries to create an admin shop (which they have no permission for). And also remember to target a chest when creating player shops via command.

            Edited May 16, 2016
